resources/lib/ps_vue.py

```diff
@@ -386,7 +394,7 @@ def get_stream(url, airing_id, channel_id, program_id, series_id, tms_id, title,
 
     r = requests.get(url, headers=headers, cookies=load_cookies(), verify=VERIFY)
     json_source = r.json()
-    stream_url = json_source['body']['video']
+    stream_url = json_source['body']['video_alt']
```

Comment:
Wouldn't it make sense to just tuck this under the if statement that checks if inputstream.adaptive is enabled? That way it will only effect the people who choose to use inputstream.adaptive.